ansible_base/authentication/utils/claims.py

```diff
@@ -905,9 +904,28 @@ def cache_existing(self, role_assignments):
             # Cache Role User Assignment
             self._init_cache_key(role_definition.name, content_type_id=role_assignment.content_type_id)
 
-            # object_id is TEXT db type
-            object_id = int(role_assignment.object_id) if role_assignment.object_id is not None else None
-            obj = role_assignment.content_object if object_id else None
+            has_valid_object_id = False
+            object_id = role_assignment.object_id  # Initialize with the original value
+
+            if role_assignment.object_id is None:
```

Comment:
this is a text field, so I'm not sure if this even works, since it might be `""`